src/core/scene/a-scene.js

```diff
@@ -465,15 +466,15 @@ module.exports = registerElement('a-scene', {
      * Renders with request animation frame.
      */
     render: {
-      value: function (time) {
+      value: function () {
         var effect = this.effect;
-        var timeDelta = time - this.time;
+        this.timer.update();
+        var time = this.timer.elapsedTime;
+        var timeDelta = this.timer.deltaTime;
 
         if (this.isPlaying) { this.tick(time, timeDelta); }
-
         this.animationFrameID = effect.requestAnimationFrame(this.render);
         effect.render(this.object3D, this.camera);
-        this.time = time;
```

Comment:
Yeah, it's documented, might be being used. And if browsers ever fix the behavior, it's possible we might revert?